arch/arm64/boot/dts/broadcom/bcm2712-rpi-cm4io.dtsi

```diff
@@ -0,0 +1,24 @@
+// SPDX-License-Identifier: GPL-2.0
+
+i2c_csi_dsi0: &i2c0 { // Note: For CAM0 and DISP0 connectors
```

Comment:
I haven't looked at the pinout, but CM5 only has 2 MIPI PHY ports, so do CAM0 and DISP0 actually connect to anything other than the I2C?

I'm sure Dominic made a comment in passing yesterday that CAM0 and DISP0 pins on the CM were repurposed for USB3.

Reply:
This is the connectivity table I ended up with:
```
CM5        CM4IO        CM5IO
---        -----        -----
DPHY0      CAM0         CAMDISP0
DPHY1      CAM1         CAMDISP1
DPHY0      DISP0        CAMDISP0
DPHY1      DISP1        CAMDISP1
GPIO0/1    CAM0, DISP0  CAMDISP1
GPIO38/39  CAM1, DISP1  CAMDISP0
```

Reply:
This means that CAM0 and DISP0 are not usable for cameras or displays, which is the better of the two options because that I2C port is effectively shared with ID_SC and ID_SD. The `i2c_csi_dsi0` parameter is there for compatibility with any existing I2C setup. `i2c_csi_dsi1` (and its alias `i2c_csi_dsi`) is assigned to CAM1 and DISP1.